File: src/main/java/de/schramm/royalbash/ServerApplication.java
package de.schramm.royalbash;

import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;

@SpringBootApplication
@ComponentScan(basePackages = {"de.schramm.royalbash"})
public class ServerApplication {

    public static final Logger logger = LogManager.getLogger(ServerApplication.class);

    public static void main(String[] args) {
        org.springframework.boot.SpringApplication.run(ServerApplication.class, args);
    }
}

File: src/main/kotlin/de/schramm/royalbash/ServerApplication.kt
package de.schramm.royalbash

import org.apache.logging.log4j.LogManager
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
class ServerApplication {
    companion object {
        @JvmStatic
        private val logger = LogManager.getLogger(ServerApplication::class.java)
    }
}

fun main(args: Array<String>) {
    org.springframework.boot.SpringApplication.run(ServerApplication::class.java, *args)
}
